lib/common/tests/operations/parse_op_key.c

```diff
@@ -164,13 +164,13 @@ empty_input(void)
     guint ms = 0;
 
     g_assert_false(parse_op_key("", &rsc, &ty, &ms));
-    g_assert_true(rsc == NULL);
-    g_assert_true(ty == NULL);
+    g_assert_null(rsc);
+    g_assert_null(ty);
```

Comment:
I didn't think we could use g_assert_null.  It was introduced in 2.38, and configure.ac has:
```
# Require glib 2.16.0 (2008-03) or later for g_hash_table_iter_init() etc.
PKG_CHECK_MODULES([GLIB], [glib-2.0 >= 2.16.0],
```

Reply:
Doh, didn't realize it was new :(

Reply:
Looks like just true/false/null/non-null are too new